app/views/students/index.html.erb

```diff
@@ -1,26 +1,41 @@
-<h1>Alunos da obra do berço</h1>
-<br>
-<table class="table">
-  <thead>
-    <tr>
-      <th scope="col">Nome</th>
-      <th scope="col">Email</th>
-      <th scope="col">Gênero</th>
-      <th scope="col">Telefone</th>
-      <th></th>
-    </tr>
-  </thead>
-  <tbody>
-    <% @students.each do |student| %>
-      <tr>
-        <td><%= student.name %></td>
-        <td><%= student.email %></td>
-        <td><%= student.gender %></td>
-        <td><%= student.mobile_phone %></td>
-        <td>
-          <%= link_to 'Editar', student_path(student) %>
-        </td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+<main>
+  <header class="header-students">
+    <h1 class="title">Alunos da Obra do Berço</h1>
+    <%= image_tag("student.svg", :alt=> "alunos da obra do berço", :class=> "img-student") %>
+  </header>
+
+  <div class="table-responsive">
+    <table class="table">
+      <thead>
+        <tr style="background-color: #adadad; border-radius: 8px 8px 0 0">
```

Comment:
What do you think of moving this css to a css file instead of using inline styles? Inline styles are not cached by the browser.

Reply:
Good idea, thanks!